.expeditor/config.yml

```diff
@@ -0,0 +1,20 @@
+product_key: inspec
+
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: compliance-notify
```

Comment:
@adamleff Wasn't sure if you wanted this to be `#inspec-notify` or `#compliance-notify`? Which ever you choose I'll be sure to sync up where the Jenkins notifications are sent.